Misha/Text/OB-185_count_words_in_a_string/count_words_test.py

```diff
@@ -0,0 +1,47 @@
+import unittest
+from count_words_in_a_string import count_words
+
+
+class CountWordsTest(unittest.TestCase):
+
+    def test_count_words_with_none_value(self):
+        with self.assertRaises(AttributeError) as context:
```

Comment:
Please download txt file for 'War and Peas' by Tolstoy and enter it as a test value!